.github/workflows/publish-prod-images.yml

```diff
@@ -73,7 +73,7 @@ jobs:
           continue-on-error: true
           with:
               context: tools/sigclient
-              file: Dockerfile
+              file: tool/sigclient/Dockerfile
```

Comment:
Why is this getting added back? I thought you just recently removed this?

Reply:
We need to give the docker file from the root path.